lib/index.js

```diff
@@ -437,7 +437,7 @@ module.exports.renderSync = function(opts) {
     return result;
   }
 
-  throw util._extend(new Error(), JSON.parse(result.error));
+  throw assign(new Error(), JSON.parse(result.error));
```

Comment:
As you say, I'm not concern with burning a couple cycle to produce an error that will kill the process anyway :)